packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -283,9 +283,9 @@ export function getReqSerializers(config: ChainForkConfig): ReqSerializers<Api,
   const getBlockReq: ReqSerializer<Api["getBlock"], GetBlockReq> = {
     writeReq: (block_id, format) => ({
       params: {block_id: String(block_id)},
-      headers: {accept: format === "ssz" ? mimeTypeSSZ : "application/json"},
+      headers: {accept: format === undefined ? MEDIA_TYPES["json"] : MEDIA_TYPES[format]},
```

Comment:
To simplify writing accept headers could add another helper function

```suggestion
      headers: {accept: writeAcceptHeader(format)},
```